medium-tests/src/test/java/mediumtest/TelemetryMediumTests.java

```diff
@@ -334,6 +336,17 @@ void it_should_record_helpAndFeedbackLinkClicked() {
       "\"helpAndFeedbackLinkClickedCount\":{\"itemId\":{\"helpAndFeedbackLinkClickedCount\":1}}"));
   }
 
+  @Test
+  void it_should_record_fixSuggestionResolved() {
+    setupClientAndBackend();
+
+    backend.getTelemetryService().fixSuggestionResolved(new FixSuggestionResolvedParams("suggestionId", FixSuggestionStatus.ACCEPTED, 0));
+    backend.getTelemetryService().fixSuggestionResolved(new FixSuggestionResolvedParams("suggestionId", FixSuggestionStatus.DECLINED, null));
```

Comment:
MINOR: I think this case is a bit unrealistic since it is unlikely the same client will send `snippetIndex` once but not another time. To be more robust, we can test the same thing with 2 different suggestion IDs.